src/capi-single-supported/index.js

```diff
@@ -62,7 +62,7 @@ function populateCard(responseJson) {
           </picture>
```

Comment:
You need to have the IE9 comments already [present](https://github.com/guardian/commercial-templates/blob/capi-multiple/src/capi-multiple-paidfor/index.html#L44) in the page in order be inserting between them with the `insertBetweenComments` function.